CONTRIBUTING.adoc

```diff
@@ -0,0 +1,267 @@
+= Spring AMQP Contributor Guidelines
+
+Have something you'd like to contribute to **Spring AMQP**?
+We welcome pull requests, but ask that you carefully read this document first to understand how best to submit them;
+what kind of changes are likely to be accepted; and what to expect from the Spring team when evaluating your submission.
+
+Please refer back to this document as a checklist before issuing any pull request; this will save time for everyone!
+
+== Code of Conduct
+This project adheres to the Contributor Covenant link:CODE_OF_CONDUCT.adoc[code of conduct].
+By participating, you  are expected to uphold this code. Please report unacceptable behavior to
+spring-code-of-conduct@pivotal.io.
+
+== Understand the basics
+
+Not sure what a *pull request* is, or how to submit one?  Take a look at GitHub's excellent documentation:
+https://help.github.com/articles/using-pull-requests/[Using Pull Requests] first.
+
+== Search JIRA first; create an issue if necessary
+
+Is there already an issue that addresses your concern?  Search the
+https://jira.springsource.org/browse/INT[JIRA issue tracker] to see if you can find something similar.
```

Comment:
Thanks; this is exactly the reason I made a PR rather than merging myself :smile: